wordpress-mobile/WordPress-iOS
# Add scaffolding for Giphy support

Implements #9974.

This PR adds a new 'Giphy' item to the `+` menu in the Media Library, with a sketelon implementation of a Giphy picker. At the moment it contains zero items, and just shows a no results view.

![giphy-1](https://user-images.githubusercontent.com/4780/44105065-6deaca38-9fe8-11e8-8ff2-5e281f6ea46e.gif)

**To test:**

* Check the code, including the feature flag
* Build and run, navigate to a site's Media Library, and tap the + button
* Choose the Giphy option, and check you see the no results view
